app/src/main/res/layout/nearby_legend.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="wrap_content"
+  android:layout_height="wrap_content"
+  android:background="#BFFFFFFF"
+  android:orientation="vertical">
+
+  <ImageView
+    android:id="@+id/imageRed"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="0dp"
+    app:layout_constraintBottom_toTopOf="@+id/imageGreen"
+    app:layout_constraintEnd_toEndOf="@+id/imageGreen"
+    app:layout_constraintStart_toStartOf="@+id/imageGreen"
+    app:srcCompat="@drawable/ic_custom_map_marker_red" />
+
+  <TextView
+    android:id="@+id/textRed"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="0dp"
+    android:text="There is a Wikidata item here with no picture yet."
```

Comment:
Would you mind making the text localizable?
See for instance `android:text="@string/app_name"` in app/src/main/res/layout/activity_about.xml

Reply:
Also, sorry to change the wording now, but I think this text would be more actionable for the user: `This place has no picture yet, go take one!`